pkg/agent/cniserver/pod_configuration.go

```diff
@@ -519,8 +515,9 @@ func (pc *podConfigurator) connectInterfaceToOVSCommon(ovsPortName string, conta
 	event := agenttypes.PodUpdate{
 		PodName:      containerConfig.PodName,
 		PodNamespace: containerConfig.PodNamespace,
-		IsAdd:        true,
 		ContainerID:  containerConfig.ContainerID,
+		NetNS:        netNS,
```

Comment:
I wonder how it handles restart case if it relies on the netNS which is not persistent anywhere. There may be two cases it can't handle:
1. If agent is restarted after CNI event is handled but before PodController handles it, it can't create no longer create the secondary interface. 
2. After agent is restarted, it can't clean up the secondary interface as the cache is always empty?

Should it cache netNS in containerConfig and use containerConfig to initialize podCache?

Reply:
Right. Today we do not persist any secondary interface configuration, and I plan to implement that in the next PR. We can definitely save net NS with the primary interface OVS port and I thought about that too. But I want to mention that only address the case antrea-agent restarts after Pod's primary interfaces are created but before any secondary interface is created, as only secondary interface creation (not deletion) will require net NS. I am still thinking is it worthwhile to cover that corner case or not.